src/uri.cr

```diff
@@ -250,26 +250,53 @@ class URI
     HTTP::Params.parse(@query || "")
   end
 
-  def to_s(io : IO) : Nil
-    if scheme
-      io << scheme
-      io << ':'
+  # Returns the authority component of this URI.
+  # It is formatted as `user:pass@host:port` with missing parts being omitted.
+  #
+  # If the URI does not have any authority information, the result is `nil`.
+  #
+  # ```cr
```

Comment:
```suggestion
  # ```
```

Reply:
@straight-shoota Please, remove this before merging.

Reply:
If someone wants to do it, we can have the formatter remove these "cr" things.

Reply:
@asterite Already done, going to make a PR soon